website/docs/reference/snapshot-configs.md

```diff
@@ -256,7 +266,7 @@ snapshots:
 
 <VersionBlock firstVersion="1.9">
 
-Configurations can be applied to snapshots using [YAML syntax](/docs/build/snapshots), available in Versionless and dbt v1.9 and higher, in the `snapshot` directory file.
+Starting from Versionless and Core v1.9, defining snapshots in a `.sql` file using a config block is a legacy method. You can define snapshots in YAML format using the latest [configurations](/docs/build/snapshots#configuring-snapshots). You can also continue using the legacy method for existing snapshots, but we recommend migrating over to the latest YAML format for better readability and maintainability.
```

Comment:
Do we want to align this section with the other ones above? Or do we want it to be slightly different?

If we want it to be the same, then we'd update it like this:
```suggestion
Starting from Versionless and Core v1.9, defining snapshots in a `.sql` file using a config block is a legacy method. You can define snapshots in YAML format using the latest [snapshot-specific configurations](/docs/build/snapshot-configs#snapshot-specific-configurations). For new snapshots, we recommend using these latest configs. If applying them to existing snapshots, you'll need to migrate over.
```